src/cmd-build

```diff
@@ -403,15 +399,7 @@ mkdir -p "${builddir}"
 mv -T "${tmp_builddir}" "${builddir}"
 # Replace the latest link
 ln -Tsf "${buildid}" builds/latest
-# Update builds.json
-# the variables passed to `prune_builds` end up single quoted and
-# python treats them as literals, so we workaround this by duplicating
-# the command  ¯\_(ツ)_/¯
-if [ "${SKIP_PRUNE}" == 1 ]; then
-    "${dn}"/prune_builds --workdir "${workdir}" --insert-only "${buildid}"
-else
-    "${dn}"/prune_builds --workdir "${workdir}"
-fi
+"${dn}"/bump-timestamp --workdir "${workdir}"
```

Comment:
We still want to prune here if `--skip-prune` wasn't passed.

Reply:
Wouldn't it be safer to be done in a separate task? Prune output would be hidden between two large outputs of build and buildextend commands

Reply:
Hmm, yeah we definitely have a logging problem (https://github.com/coreos/coreos-assembler/issues/692), though that seems like a mostly separate issue from whether to keep the current behaviour of automatically pruning after every build, right?

Reply:
Okay, makes sense. Added this in 82a338e7df8ea614daa43a8acf842bc086352b90